app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -659,6 +659,8 @@ class BrowserTabFragment : Fragment(), FindListener, CoroutineScope {
 
             it.setFindListener(this)
         }
+
+        WebView.setWebContentsDebuggingEnabled(BuildConfig.DEBUG)
```

Comment:
Optional: When we optimize then this code could disappear if we use this format
```
if (BuildConfig.DEBUG) {
   WebView.setWebContentsDebuggingEnabled(true)
}
```

Reply:
i'm not sold on the (premature) optimization angle, but i think your suggestion is more readable. so we agree - but for different reasons 😄